client/src/components/ViewProfile.js

```diff
@@ -35,150 +22,197 @@ const ViewProfile = () => {
-                  </Header>
-                  <Card.Content>
-                    <Label>Groups:</Label>
-                    <p>{profile.groups && profile.groups.join(", ")}</p>
-                    <Label>How did they hear about us:</Label>
-                    <p>{profile.how_did_they_hear}</p>
-                    <Label>they joined:</Label>
-                    <p>
-                      {new Date(profile.join_date).toString().substring(4, 15)}
-                    </p>
-                    <Label>Notes:</Label>
-                    <p>{profile.notes}</p>
-                  </Card.Content>
-                </Card.Content>
-              </Card>
-            </Grid.Column>
-          </Grid.Row>
-          <Modal open={modalStatus} size="small">
-            <Header content="!Delete this profile" />
-            <Modal.Content>
-              <p>
-                Are you sure you want to delete this profile? Once deleted, it
-                cannot be recovered.
-              </p>
-            </Modal.Content>
-            <Modal.Actions>
-              <Button basic negative onClick={() => setModalStatus(false)}>
-                <Icon name="cancel" /> No
-              </Button>
-              <Button
-                basic
-                positive
-                onClick={() => {
-                  setDelProfile(true);
-                }}
+      <div className="flex flex-col max-w-5xl mx-auto">
+        <div className="flex justify-between">
+          <div>
+            <a
+              className="group flex items-center text-xl font-semibold cursor-pointer"
+              onClick={() => history.goBack()}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                className="text-blue-300 mr-3 h-8 w-8"
+                viewBox="0 0 20 20"
+                fill="currentColor"
               >
-                <Icon name="check" /> Yes
-              </Button>
-            </Modal.Actions>
-          </Modal>
-        </Grid>
-      </Segment>
+                <path
+                  fillRule="evenodd"
+                  d="M10 18a8 8 0 100-16 8 8 0 000 16zm.707-10.293a1 1 0 00-1.414-1.414l-3 3a1 1 0 000 1.414l3 3a1 1 0 001.414-1.414L9.414 11H13a1 1 0 100-2H9.414l1.293-1.293z"
+                  clipRule="evenodd"
+                />
+              </svg>
+              Go back
+            </a>
+          </div>
+          <div>
+            <button className="btn px-6 py-3" disabled>
+              Edit
+            </button>
+            <button
+              className="btn px-4 py-3 ml-4"
+              onClick={() => setShowDeleteModal(true)}
+            >
+              Delete
+            </button>
+          </div>
+        </div>
+        <div>
+          <div className="flex justify-left items-center mt-10 space-x-5">
+            <h1 className="text-2xl font-bold">{`${profile.first_name} ${profile.last_name}`}</h1>
+            <div className="text-lg">{profile.status}</div>
```

Comment:
For the status I would keep the same layout as the profile list, with different color depending on the status.  
Maybe we can move the function getClassName to an helper file so that it's available for both components.

Reply:
Yes, it would be better if it used the same colour codes. There are a lot of issues like that though (the original version used the yellow colour for all the profiles). This PR was mainly about getting rid of Semantic UI so where other stuff was broken I just left it in a basic state rather than trying to fix everything. Some of the other information in the profile view doesn't show at all (like the nationality), so there is plenty of scope for improvement.